servers/display_server.cpp

```diff
@@ -1307,12 +1307,11 @@ bool DisplayServer::can_create_rendering_device() {
 	}
 #endif
 #ifdef METAL_ENABLED
-	if (rcd == nullptr) {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wunguarded-availability"
-		// Eliminate "RenderingContextDriverMetal is only available on iOS 14.0 or newer".
-		rcd = memnew(RenderingContextDriverMetal);
-#pragma clang diagnostic pop
+	if (__builtin_available(macOS 11.0, iOS 14.0, *)) {
+		if (rcd == nullptr) {
+			// Eliminate "RenderingContextDriverMetal is only available on iOS 14.0 or newer".
+			rcd = memnew(RenderingContextDriverMetal);
```

Comment:
The comment here related to the diagnostic wrappers. Assuming that's now dealt with via the `__builtin_available` check, it can be removed
```suggestion
			rcd = memnew(RenderingContextDriverMetal);
```

Reply:
Indeed, it is comment for "-Wunguarded-availability" and no longer needed.